app/src/androidTest/java/com/github/se/project/ui/profile/EditProfileTest.kt

```diff
@@ -207,28 +207,6 @@ class EditProfileTest {
     Mockito.verify(mockNavigationActions, never()).navigateTo(Mockito.anyString())
   }
 
-  /*@Test
-  fun editNameButton() {
-    (mockViewModel.currentProfile as MutableStateFlow).value = profile
-    // Set the screen in the test environment
-    composeTestRule.setContent {
-      EditProfile(navigationActions = mockNavigationActions, mockViewModel)
-    }
-    composeTestRule.onNodeWithTag("editNameButton").performClick()
-    composeTestRule.onNodeWithTag("firstNameField").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("lastNameField").assertIsDisplayed()
-
-    composeTestRule.onNodeWithTag("firstNameField").performTextInput("New")
-    composeTestRule.onNodeWithTag("lastNameField").performTextInput("New")
-
-    composeTestRule.onNodeWithTag("confirmButton").performClick()
-
-    assertEquals("NewFirst", mockViewModel.currentProfile.value?.firstName)
-    assertEquals("NewLast", mockViewModel.currentProfile.value?.lastName)
-
-    verify(mockNavigationActions).goBack()
-  }*/
```

Comment:
Like before, if you delete this test because it add commented lines to the main, we should put it in the shared file so that we do not loose it and we can address the solving of the bug next week.

Reply:
In this case, this is a test for a feature that was removed a bit ago: The ability to edit the profile's name.

I personally would like to see that feature come back, but as is this test doesn't have a purpose